src/components/activities/adminActivity/SideMenu/SideMenu.tsx

```diff
@@ -102,6 +105,21 @@ const SideMenu = () => {
                     <span className={classes.value}>{entry[1] ?? "---"}</span>
                   </div>
                 ))}
+            <Button
+              type="button"
+              variant="text"
+              dataCy="edit-hospital"
+              sx={{
+                color: "black",
+                backgroundColor: "white",
+                "&:hover": { backgroundColor: "gray" },
+              }}
```

Comment:
Isn't it possible to do this with CSS classes rather than introducing this property?